projects/devices/slamtec/protos/RpLidarA2.proto

```diff
@@ -58,7 +58,7 @@ PROTO RpLidarA2 [
       density -1
       mass 0.2
     }
-    fieldOfView 1.57
+    fieldOfView 6.283184
     horizontalResolution 800
```

Comment:
```suggestion
    horizontalResolution 400
```

The resolution is 0.9°. So if the field of view is 360° (6.28 rad), it should be 400 here.

Reply:
@ygoumaz I would like to have the same frequency define in .proto file and LaserScan topic in ROS.

I found the bug when the `updateRate` is not define the LaserScan is published with 30Hz but the period should be 0 [here](https://github.com/cyberbotics/webots_ros2/blob/master/webots_ros2_driver/src/plugins/Ros2SensorPlugin.cpp#L26).

I modified Ros2Lidar to unify .proto and ROS frequency. https://github.com/cyberbotics/webots_ros2/pull/674 
The `defaultFrequency 12` gives:
```
average rate: 10.266
        min: 0.087s max: 0.106s std dev: 0.00246s window: 5611
```
using `ros2 topic hz /scan`